packages/fxa-settings/src/components/TermsPrivacyAgreement/en.ftl

```diff
@@ -6,6 +6,7 @@ terms-privacy-agreement-intro = By proceeding, you agree to:
 # links to Pocket's Terms of Service and Privacy Notice
 terms-privacy-agreement-pocket = { -product-pocket }’s <pocketTos>Terms of Service</pocketTos> and <pocketPrivacy>Privacy Notice</pocketPrivacy>
 # links to Firefox's Terms of Service and Privacy Notice
+## TODO (before PR merge): Does this need a new ID w/ { -brand-mozilla }?
```

Comment:
I'll push an update here and on some related strings before taking this out of draft.